packages/server/test/migration/global-diff.test.ts

```diff
@@ -9,13 +11,14 @@ describe('Global Diff', () => {
   test(
     'Starts Messaging with the latest database schema',
     async () => {
+      const port = await portfinder.getPortPromise()
       await startMessagingServer(
         {
```

Comment:
I think we should use portfinder on all of our tests. I regularly run the tests when some instance of messaging is running at it forces be to close that instance of messaging